WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Shipping Address Validation/ShippingLabelAddressFormViewController.swift

```diff
@@ -266,6 +266,9 @@ extension ShippingLabelAddressFormViewController: UITableViewDelegate {
         switch row {
         case .state:
             let states = viewModel.statesOfSelectedCountry
+            guard !states.isEmpty else {
```

Comment:
nit: here you can use directly `isNotEmpty` 👍

Reply:
Thanks for the suggestion! I didn't know about this before 👍